polaris-react/src/components/Tabs/Tabs.scss

```
@@ -111,6 +119,35 @@
   &[aria-disabled='true'] path {
     fill: var(--p-color-icon-disabled);
   }
+
+  #{$se23} & {
+    color: var(--p-color-text-primary);
```

Think we also need to override the original 4px border-radius here so that it matches the 8px in the designs

```suggestion
    border-radius: var(--p-space-2);
    color: var(--p-color-text-primary);
```